clientapp/src/utilities/mapUtilities.js

```diff
@@ -112,13 +116,15 @@ export const BingMap = {
             searchManager.geocode(requestOptions)
         })
     },
-    drawThePinByGeocoords:function(lat, lon, message) { // Draw the pin
+    drawThePinByGeocoords:function(lat, lon, message, shouldSetView) { // Draw the pin
         window.Microsoft.Maps.loadModule('Microsoft.Maps.Search', function () {
             var searchManager = new window.Microsoft.Maps.Search.SearchManager(map);
             var reverseGeocodeRequestOptions = {
                 location: new window.Microsoft.Maps.Location(lat,lon),
-                callback: function (answer, userData) {
-                    map.setView({ bounds: answer.bestView })
+                callback: function (answer, userData, shouldSetView) {
```

Comment:
don't want to pass this into the callback. the callback will not be called with a 3rd parameter so this will always be undefined. If you want to use the shouldSetView from above, you can just call it directly. On the other hand, since these functions are only being called with shouldSetView=false, I wonder if we should just leave it out and not set the view within these functions.

Reply:
Have removed the shouldSetView